Tribler/dispersy

```diff
@@ -1 +1 @@
-Subproject commit fec55be1708ac9fd70bdde9714f71e58f88d7697
+Subproject commit 2c6efd6d2661f96dc0180d0acdc96afac5590382
```

Comment:
Strange, I think it's older? 
See https://github.com/Tribler/dispersy/commits/devel